scripts/ci/install:build

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
colon in filenames are invalid on windows. It is understandable that files under `scripts/ci/*` are meant for ci only. 

However, on windows, with git, these files get removed and creates clutter. Would you be okay to rename files with colon to underscore maybe?

Reply:
I would be -1 on underscores, I want it to be visually evident that the there is a component:subcomponent relationship in these files and I don't think an underscore would convey that as it would just look like a snake case name. Possible alternatives:

* Use other unicode characters that happen to be similar to the ascii colon: 
https://stackoverflow.com/questions/10386344/how-to-get-a-file-in-windows-with-a-colon-in-the-filename

* Use other ascii punctuation, `install.build`, `install~build`...

Reply:
+1 for using any of `install.build`, `install~build`  ...

Reply:
I would be wary about using `~`. In fact, `:` is problematic in some shells on linux as well (e.g. I don't have auto-completion with `:` in file names). `.` perhaps the safest.

Reply:
@pratapvardhan please feel free to submit a PR with a change to `.`